NHLBI_BioData_Catalyst/Rstudio/ORCHID_COVID19.Rmd

```diff
@@ -82,16 +82,22 @@ All these outcomes will be analyzed using multivariable models, taking into acco
 
 # Packages Installation 
 
+**Note that if you are using the dedicated PIC-SURE environment within the BioData Catalyst Seven Bridges platform, the necessary packages have already been installed.**
+
 ```{r}
-source("Rstudio_lib/ORCHID_requirements.R")
+### Uncomment this code if you are not using the PIC-SURE environment in Seven Bridges, or if you do not have all the necessary dependencies installed.
+#source("Rstudio_lib/ORCHID_requirements.R")
```

Comment:
For SB, I cannot run the code without installing the packages, so I need to run this line. However, when I try to run it, I get the following error:
`Warning in install.packages :
  unable to access index for repository https://mran.microsoft.com/snapshot/2022-07-29/src/contrib:
  cannot open URL 'https://mran.microsoft.com/snapshot/2022-07-29/src/contrib/PACKAGES'
Installing package into ‘/usr/local/lib/R/site-library’
(as ‘lib’ is unspecified)
Warning in install.packages :
  unable to access index for repository https://mran.microsoft.com/snapshot/2022-07-29/src/contrib:
  cannot open URL 'https://mran.microsoft.com/snapshot/2022-07-29/src/contrib/PACKAGES'
Warning in install.packages :
  package ‘versions’ is not available for this version of R

A version of this package for your version of R might be available elsewhere,
see the ideas at
https://cran.r-project.org/doc/manuals/r-patched/R-admin.html#Installing-packages
Error in library(versions) : there is no package called ‘versions’`

When trying to install the packages separately, I ran into additional errors like "package doesn't exist" (specifically for arsenal, needed for tableby function).

Reply:
Note: This line of code also errors when testing in Terra

`Error: package or namespace load failed for ‘tidyverse’:
.onAttach failed in attachNamespace() for 'tidyverse', details:
call: value[[3L]](cond)
error: Package ‘tibble’ version 3.1.7 cannot be unloaded:
Error in unloadNamespace(package) : namespace ‘tibble’ is imported by ‘tidyr’, ‘modelr’, ‘haven’, ‘ggplot2’, ‘rstatix’, ‘dplyr’, ‘readr’, ‘dbplyr’, ‘broom’, ‘ggpubr’, ‘rio’ so cannot be unloaded`

Reply:
Should be resolved in SB

Reply:
Terra should be resolved in [18b78eef9ff32312c11539fcc0885482d7897e23](https://github.com/hms-dbmi/Access-to-Data-using-PIC-SURE-API/commit/18b78eef9ff32312c11539fcc0885482d7897e23)